verilog/tools/ls/symbol-table-handler.cc

```diff
@@ -248,7 +251,43 @@ std::vector<verible::lsp::Location> SymbolTableHandler::FindDefinitionLocation(
   const verible::TextStructureView &text = parsedbuffer->parser().Data();
 
   const verible::TokenInfo cursor_token = text.FindTokenAt(cursor);
-  auto symbol = cursor_token.text();
+  return cursor_token.text();
+}
+
+std::optional<verible::lsp::Location>
+SymbolTableHandler::GetLocationFromSymbolTableNode(
+    absl::string_view symbol, const VerilogSourceFile *file_origin) {
```

Comment:
the word `symbol` is confusing here as it is often usef for a `verible::Symbol*`. Maybe `symbol_name` ?

Reply:
Fixed